### ISSUE TITLE
Bower package not found

### ISSUE THREAD

**Issue by OzzieOrca** [eknowles/angular-color-this#1]
When running `bower install angular-color-this --save`, bower is unable to find the package:

```
bower ENOTFOUND     Package angular-color-this not found
```

Could you register the package again? Thanks!

**Comment by JimmyFarcy** [comment]
I agree, i try it today it was not available, and using the basic way to get it (git clone) i can't clone it

**Comment by elvismacak** [comment]
Yeah, the same problem.
so I've to use `bower install https://github.com/eknowles/angular-color-this.git#v1.0.1  --save`, and this works for me

**Comment by eknowles** [comment]
Updating it today sorry guys!

**Comment by Slauta** [comment]
Hi, fix commit merge #4
